README.md

```diff
@@ -32,6 +32,77 @@ def deps do
 end
 ```
 
+### Installation as standalone service
+
+For now there are two ways of starting `fennec` as standalone application. Via release build from
+source or via prebuild docker image. The docker image could be used for production system with a proper
+network setup (the easiest one would be `--net=host`). For developement, on non-docker-native platforms
+it probably is simpler to start the builded release then setup docker container to work correclty.
+This is due to the fact that TURN server allocates peer ports for system ephemeral ports pool which is
```

Comment:
Maybe `TURN server uses system ephemeral port range/pool for allocations?` ? What do you think?